### DISCUSSION TITLE
Could not connect to Radarr error

### DISCUSSION THREAD

**Discussion by mag1c22** [mza921/Plex-Auto-Collections#241] [Q&A]
Hi all setup my config per the readme and this is what I get:

![image](https://user-images.githubusercontent.com/94148902/141386264-0a77ec34-49f6-45a5-bf2e-674cb7e76feb.png)

I am on Version 3.2.2.5080 of radarr and the API key is valid and so is the IP address. I am also running this on unraid for what it's worth.

Clearly I am missing something, any ideas?

Config:

radarr:
      url: http://192.168.1.246:7878/radarr/
      version: v3
      token: #####
      quality_profile_id: 4
      root_folder_path: /mnt/movies
      add_to_radarr: false
      search_movie: false

**Comment by meisnate12** [comment]
development here has stalled check out [Plex Meta Manager](https://github.com/meisnate12/Plex-Meta-Manager) its a complete rewrite of Plex Auto Collections. It has many many bug fixes and other feature improvements.

As for your issue it would probably also happen with PMM i think the url you need to get it working is just `http://192.168.1.246:7878`
